### PR TITLE
[DUOS-1918][risk=no] font/ui changes to voting pages

### PR DESCRIPTION
**Pull request by shaemarks**
Addresses [DUOS-1918](https://broadworkbench.atlassian.net/browse/DUOS-1918)
Changes:
(Notes from Miro)
- Add title "Final Vote Summary" in lightweight grey above `DataUseVoteSummary` header 
- Change "RP" to "RUS" in RP Vote `VoteResultBox` of `DataUseVoteSummary` header 
- Change "Research Proposal" heading of `MultiDatasetVotingTab` to "Research Use Statement"
- Change "Structured Research Purpose" on `ResearchProposalVoteSlab` to "Research Use Statement (GA4GH DUO)"
- Remove "Primary" and "Secondary" labels to data use pills 
- Change "Research Purpose" subheading of `ResearchProposalVoteSlab` to Research Use Statement (Narrative)
- Remove 'Your Vote' text above voting buttons 
- Add "Enter your comments prior to voting" to rationale text box 
- Add: "Datasets Requested by Data Use" above dataset slabs in same format as 'Research Proposal' from the top of the page 

Changes not on Miro board comments:
-  changed dataCy fields containing srp to rp since "Structured Research Purpose" is no longer a term being used 
- moved function that mapped data uses to data use pills from `ResearchProposalVoteSlab` file to `DataUsePill` file since used by both `ResearchProposalVoteSlab` and `MultiDatasetVoteSlab` components 
- use rpVoteKey consistently rather than comparing against a string 

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/components/collection_vote_box/CollectionSubmitVoteBox.js [hunk 1]
```diff
@@ -152,7 +151,7 @@ export default function CollectionSubmitVoteBox(props) {
           textarea({
             name: 'Rationale Input',
             value: rationale,
-            placeholder: 'Optional: Describe your rationale or add comments here',
+            placeholder: 'Optional: Describe your rationale or add comments here. Enter your comments prior to voting.',
```
Comment (connorlbark) [inline src/components/collection_vote_box/CollectionSubmitVoteBox.js]: only thought is that this seems too long; maybe `Optional: Enter your comments and describe your rationale prior to voting.` not sure if this is inline with the original feedback, though.

**Review by connorlbark — APPROVED** [review]
had some extra time, so i thought i would go ahead and look over this today
checked off each of the feedback items, and they all were covered in the UI. minor optional correction but it looks good to me!

File: src/components/collection_vote_box/CollectionSubmitVoteBox.js (hunk 1 above)
Comment (shaemarks) [reply src/components/collection_vote_box/CollectionSubmitVoteBox.js]: Good suggestion!

**Review by rushtong — APPROVED** [review]
👍🏽

**Pull request merged**